packages/kbn-visualization-ui-components/components/chart_switch_trigger.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
```

Comment:
> - [ ] Also per the [Google doc comments](https://docs.google.com/document/d/1XVRxJOyXJ1QcuX0hRP0mzAVjDId1TjfrSCBB-55kv00/edit?disco=AAABGwTt9Qo), I agree that it's a bit problematic that deleting a required dimensions doesn't properly inform the user. Would it be possible to show the standard Lens "Requires field" validation message in red as we typically do for users that outright deleted required dimensions?
>   
>   ![CleanShot 2024-03-06 at 12 31 58](https://github.com/elastic/kibana/assets/3884767/252baf62-1af4-4a2a-91e2-b8bd6d216ef2)


We still do it, we just don't do it when the visualization is completely cleared out. Do you think we should change it? It's possible, but I'd do it separately 🙏🏼

Reply:
Yeah, I think it'd be a better user experience to show that sort of "required" error on the appropriate dimensions even when the entire configuration is cleared out. I'm fine with splitting it out as a separate issue as well.

Reply:
Done: https://github.com/elastic/kibana/issues/178237